.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 14 * * *'
```

Comment:
Can you please allow manual triggers too? Example: https://github.com/elastic/elasticsearch-benchmarks/blob/ce628605c99b4dedccbbe13c74bdfc841e9b071b/.github/workflows/update-visualizations.yml#L6-L7

Reply:
Done.